integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseOpenUtils.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse;
+
+import jnr.constants.platform.OpenFlags;
+
+/**
+ * Alluxio Fuse utilities to handle different fuse.open() flags.
+ * Decide Alluxio open action based on fuse open flags.
+ */
+public final class AlluxioFuseOpenUtils {
+  // TODO(lu) change to bit operation to speed up
+  // Extra commonly used Fuse open flags other than defined in
+  // {@link jnr.constants.Constant.OpenFlags}
+
+  // r Open file for reading. An exception occurs if the file does not exist.
+  // (Decimal): 32768 (Octal): 00000100000 (Hexadecimal): 00008000
+  // (Binary ): 00000000000000001000000000000000
+  private static final int READ =  32768;
```

Comment:
Could you use hex number for these flag? Something like 

```java
private static final int READ =  0x8000;
private static final int R_OR_W_PLUS =  0x8002;
```

So that we can have a sense of the flag bit

Reply:
Changed to hex, PTAL